Makefile

```diff
@@ -111,3 +111,7 @@ dcshell:
 dctest: endtoendtest
 	# launch all studio's dependent services using docker-compose, and then run the tests
 	echo "Finished running  make test -e DJANGO_SETTINGS_MODULE=contentcuration.test_settings"
+
+dcservicesup:
```

Comment:
I think it would be nice to also add:
```
dcservicesdown:
	docker-compose -f docker-compose.yml -f docker-compose.alt.yml down
```

Since we're overriding and merging the two yamls, there is a potential that `dcdown` may not function correctly after `dcservicesup`.